patches/@types+react-native+0.57.65.patch

```diff
@@ -0,0 +1,280 @@
+diff --git a/node_modules/@types/react-native/globals.d.ts b/node_modules/@types/react-native/globals.d.ts
```

Comment:
Do we need that file?

Reply:
Yeah it is a path for react-native because:
>after import gutenberg-mobile we have react-native-dark-mode as a dependency which requires @types/react-native. `@types/react-native/globals.d.ts` and `typescript/lib/lib.dom` declare the same global types. We can not remove `lib.dom` from `tsconfig` because `@types/react-native` doesn't include Window and i get error:
```
node_modules/@types/reach__router/index.d.ts:9:30 - error TS2304: Cannot find name 'Window'.

9 export type WindowLocation = Window['location'] & HLocation;
```
>I tried all solutions from this issue (https://github.com/DefinitelyTyped/DefinitelyTyped/issues/15960)  and few more, only removing the `@types/react-native/globals.d.ts` fix that.

Seems like Orta is trying to fix that here - https://github.com/microsoft/types-publisher/pull/655